### ISSUE TITLE
simple: Suggest replacing bytes.Buffer.Truncate(0) with Reset()

### ISSUE THREAD

**Issue by dlsniper** [dominikh/go-tools#1080]
[Per documentation](https://pkg.go.dev/bytes#Buffer.Reset), the following code:
```go
buff := bytes.NewBuffer(nil)
buff.Truncate(0)
```

can be expressed simpler as:
```go
buff := bytes.NewBuffer(nil)
buff.Reset()
```

**Comment by dominikh** [comment]
I don't think this is worth a check in `simple`. It doesn't make the code objectively better, and it is prone to noisy positives with surrounding code. I'm not sure it's even worth a quickfix. `Reset` isn't any clearer than `Truncate(0)` IMO.
